packages/addon-dev/sample-rollup.config.js

```diff
@@ -1,15 +1,28 @@
-import babel from '@rollup/plugin-babel';
+import { babel } from '@rollup/plugin-babel';
+import { nodeResolve } from '@rollup/plugin-node-resolve';
+
 import { Addon } from '@embroider/addon-dev/rollup';
 
+import packageJson from './package.json';
```

Comment:
Hmm, what makes this work? By default you can't import from `.json` files in Node...

Reply:
Rollup configs go through their own weird preprocessing. That's also why they're written as ES modules despite predating Node's native ES module support.